Docs: Tracing Instrumentation doesn't work for outgoing axios requests in nodejs function runtimes (#13696)
kyma-project/kyma: docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

   >**NOTE:** The `sap.kyma.custom.acme.payload.sanitised.v1` is a sample event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](../../05-technical-reference/evnt-01-event-names.md) page. The receiver subscribes to the event type to consume the events.

   >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](../../05-technical-reference/svls-08-function-specification.md#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.

>    >**NOTE:** Please note there is a `require('axios')` line even though the function code is not using it directly. This is needed for the auto-instrumentation to properly handle outgoing requests sent via `publishCloudEvent` method (which uses `axios` library under the hood). Without `axios` import the function would still work, but the published events would not be reflected in Jaeger.

Reviewer: ```suggestion
   >**NOTE:** There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in Jaeger.
```
